components/kyma-environment-broker/internal/storage/dbsession/write.go

```diff
@@ -7,13 +7,46 @@ import (
 	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal"
 	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage/dberr"
 	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage/postsql"
+	"github.com/lib/pq"
+)
+
+const (
+	UniqueViolationErrorCode = "23505"
 )
 
 type writeSession struct {
 	session     *dbr.Session
 	transaction *dbr.Tx
 }
 
+// OperationType defines the possible types of an asynchronous operation to a broker.
+type OperationType string
+
+const (
+	// OperationTypeProvision means provisioning OperationType
+	OperationTypeProvision OperationType = "provision"
+	// OperationTypeDeprovision means deprovision OperationType
+	OperationTypeDeprovision OperationType = "deprovision"
+	// OperationTypeUndefined means undefined OperationType
+	OperationTypeUndefined OperationType = ""
+)
+
+type OperationDTO struct {
```

Comment:
Why this struct is here? not with others in the `internal` (model.go) package? `internal.Operation` cannot be used here?

Reply:
It is about how to store in the DB, it needs Type and Data